src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,22 @@
+import './Gallery.scss'
+
+type GalleryProps = { galleryArray: string[] }
+
+const Gallery = ( { galleryArray }: GalleryProps) => {
+
+const gallery = galleryArray ?? ["Sorry! Image not found."]
```

Comment:
I am not sure about this

`const gallery = galleryArray ?? ["Sorry! Image not found."]`

We would probably want the Event component to check the length of the image array and display the Gallery component based on that rather than check it in the component.